src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  } else if (days < 3 || days > 7) {
+    return days * 40;
```

Comment:
Replace the literals in expressions with named constants to make the code more readable and maintainable. For example, instead of using '40', '50', and '20', define constants like 'DAILY_RATE', 'LONG_TERM_DISCOUNT', and 'SHORT_TERM_DISCOUNT'.